content/tutorials/manage-account-settings/use-tickets.md

```diff
@@ -0,0 +1,372 @@
+{{ end_step() }}
+
+
+### {{n.next()}}. Check Sequence Number
+
+If you already have at least one Ticket available in the ledger, you can skip this step. Otherwise, you need to get ready to create some Tickets for your other transactions to use.
+
+Before you create any Tickets, you should check what [Sequence Number][] your account is at. You want the current Sequence number for the next step, and the Ticket Sequence numbers it sets aside start from this number.
+
+```js
+async function get_sequence() {
+  const account_info = await api.request("account_info", {
+      "account": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe"
+  })
+  console.log("Current sequence:", account_info.account_data.Sequence)
+  return account_info.account_data.Sequence
+}
+
+let current_sequence = get_sequence()
+```
+
+{{ start_step("Check Sequence") }}
+<button id="check-sequence" class="btn btn-primary connection-required"
+  title="Complete all previous steps first" disabled>Check Sequence Number</button>
+<div id="check-sequence-output"></div>
+{{ end_step() }}
+
+
+
+### {{n.next()}}. Prepare and Sign TicketCreate
+
+If you already have at least one Ticket available in the ledger, you can skip this step. Otherwise, you need to prepare the transaction to create some Tickets.
+
+Construct a [TicketCreate transaction][] the sequence number you determined in the previous step. Use the `TicketCount` field to set how many Tickets to create. For example, this example [prepares a transaction](rippleapi-reference.html#preparetransaction) that would make 10 Tickets:
+
+```js
+let prepared = await api.prepareTransaction({
+  "TransactionType": "TicketCreate",
+  "Account": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+  "TicketCount": 10,
+  "Sequence": current_sequence
+})
+console.log("Prepared transaction:", prepared.txJSON)
+let max_ledger = prepared.instructions.maxLedgerVersion
+
+let signed = api.sign(prepared.txJSON, "s████████████████████████████")
+console.log("Transaction hash:", signed.id)
+
+let tx_blob = signed.signedTransaction
+```
+
+Take note of the transaction's hash and `LastLedgerSequence` value so you can [be sure whether or not it got validated](reliable-transaction-submission.html) later.
+
+
+{{ start_step("Prepare & Sign") }}
+<button id="prepare-and-sign" class="btn btn-primary connection-required"
+  title="Complete all previous steps first" disabled>Prepare & Sign</button>
+<div id="prepare-and-sign-output"></div>
+{{ end_step() }}
+
+
+
+### {{n.next()}}. Submit TicketCreate
+
+If you already have at least one Ticket available in the ledger, you can skip this step. Otherwise, you need to send a transaction to create some Tickets.
+
+Submit the signed transaction blob that you created in the previous step. Take note of the latest validated ledger index at the time of submission, so you can set a lower bound on what ledger versions the transaction could be validated in. For example:
+
+```js
+let prelim_result = await api.request("submit", {"tx_blob": tx_blob})
```

Comment:
nit: As of ES6 (aka ES2015) we can use a [shorthand property name](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer):
```suggestion
let prelim_result = await api.request("submit", {tx_blob})
```

Reply:
That's fascinating, and totally new to me. I think the wordier traditional syntax might be a bit more obvious/clear for people who aren't familiar with this new piece of syntactic sugar. Seems like it would be easy to get confused or mix it up (forget the braces, rename the variable and not understand why it's not working) if you're not familiar with the idea.